src/Features/CSharp/Portable/ReplacePropertyWithMethods/CSharpReplacePropertyWithMethodsService.cs

```diff
@@ -305,6 +294,41 @@ public override SyntaxNode GetPropertyNodeToReplace(SyntaxNode propertyDeclarati
             return propertyDeclaration;
         }
 
+        protected override NameMemberCrefSyntax TryGetCrefSyntax(IdentifierNameSyntax identifierName)
+        {
+            return identifierName.Parent as NameMemberCrefSyntax;
+        }
+
+        protected override NameMemberCrefSyntax CreateCrefSyntax(NameMemberCrefSyntax originalCref, SyntaxToken identifierToken, SyntaxNode parameterType)
+        {
+            CrefParameterListSyntax parameterList;
+            if (parameterType is TypeSyntax)
+            {
+                CrefParameterSyntax parameter = SyntaxFactory.CrefParameter((TypeSyntax)parameterType.ReplaceTokens(parameterType.DescendantTokens(), ReplaceBraceTokenForDocumentation));
+                parameterList = SyntaxFactory.CrefParameterList(SyntaxFactory.SingletonSeparatedList(parameter));
+            }
+            else
+            {
+                parameterList = SyntaxFactory.CrefParameterList();
+            }
+
+            return SyntaxFactory.NameMemberCref(SyntaxFactory.IdentifierName(identifierToken), parameterList);
+        }
+
+        private static SyntaxToken ReplaceBraceTokenForDocumentation(SyntaxToken originalToken, SyntaxToken rewrittenToken)
+        {
+            switch (rewrittenToken.Kind())
+            {
+                case SyntaxKind.LessThanToken:
+                case SyntaxKind.GreaterThanToken:
+                    string newText = rewrittenToken.IsKind(SyntaxKind.LessThanToken) ? "{" : "}";
```

Comment:
The source node is generated by `SyntaxGenerator.TypeExpression`. If there's an equivalent method which I can use that knows we're in a documentation comment I could use that instead.